.TAOS-CI/taos/menu.html

```diff
@@ -0,0 +1,48 @@
+<html>
```

Comment:
Why this file is being installed in nnstreamer.git?
Isn't this supposed to be in TAOS-CI.git?

Reply:
Becaust this feature is only for nnstreamer.